tests/test_calendars.py

```diff
@@ -110,3 +111,48 @@ def test_visakha_bucha_date(self):
                 visakha_bucha_year_date[year],
                 self.calendar.visakha_bucha_date(year),
             )
+
+
+class TestRelativeWeekdays(unittest.TestCase):
+    def test_weekday_of_month(self):
+        # 1st Monday of 2023 months
+        for month, day in enumerate(
+            (
+                3,
+                7,
+                7,
+                4,
+                2,
+                6,
+                4,
+                1,
+                5,
+                3,
+                7,
+                5,
+            ),
+            1,
```

Comment:
and this should help with the "platforming"
```suggestion
            1
```

Reply:
Wow, that's great! I should do this trick in Malaysia test!

Reply:
Done